chromadb/db/impl/grpc/server.py

```diff
@@ -318,6 +318,7 @@ def CreateCollection(
             dimension=request.dimension,
             database=database,
             tenant=tenant,
+            version=None,
```

Comment:
Can we call it collection_version?

Reply:
No. Its not called that anywhere else

Reply:
https://github.com/chroma-core/chroma/blob/6c088cdbbb09fb94c266f26bd063761fdc175370/idl/chromadb/proto/chroma.proto#L59